contrib/hook.sh

```diff
@@ -4,10 +4,5 @@ echo "WASP SWAP hook"
 
 set -x
 
-CG_PATH=$(jq -er '.linux.cgroupsPath' < config.json)
-
-if [[ "$CG_PATH" = *burst* ]];
-then
-  CONTAINERID=$(jq -er '.linux.cgroupsPath | split(":")[2]' < config.json)
-  echo max > /sys/fs/cgroup/kubepods.slice/*burst*/*/crio-$CONTAINERID*/memory.swap.max
-fi
+CONTAINERID=$(jq -er '.linux.cgroupsPath | split(":")[2]' < config.json)
+runc update $CONTAINERID --memory-swap -1
```

Comment:
Will this

1. change the current configuration?
2. change the persisted configuration?

Reply:
Change the persisted configuration

Reply:
Sorry I meant to say that when you initiate the runc udpate, changes are taking effect right away and they also persist.

Reply:
Ack, col.

Reply:
s/col/cool/g